File: Day32/Java/Fibonacci_Number_Leetcode.java
public class Fibonacci_Number_Leetcode {
    static int fibo(int n){
        if(n<=1) return n;
        int last = fibo(n-1);
        int slast = fibo(n-2);
        return last + slast;
    }
    static int fib(int n) {
        int ans = fibo(n);
        return ans;
    }
    public static void main(String[] args) {
        int n = 6;
        int ans = fib(n);
        System.out.println(ans);
    }

}

File: Day32/CPP/Fibonacci_Number_Leetcode.cpp
#include<bits/stdc++.h>
using namespace std;
int fibo(int n){
    if(n<=1) return n;
    int last = fibo(n-1);
    int slast = fibo(n-2);
    return last+slast;
}
int fib(int n) {
    int ans = fibo(n);
    return ans;
}
int main()
{
    int n;
    cin>>n;
    int ans  = fib(n);
    cout<<ans;
    return 0;
}